requirements-unchecked.txt

```diff
@@ -0,0 +1,2 @@
+git+git://github.com/benoitc/http-parser.git@2306df28fbd7946036a2609776a77833f5dda81e#egg=http-parser
```

Comment:
why do we need this file?

Reply:
There is no release of `http-parser` which supports Python 3.7
See e.g. https://github.com/benoitc/http-parser/issues/80 .
So, we have to compile it from `master`.

Reply:
Not saying that you need to do it in this PR, but that project seems unmaintained. https://github.com/MagicStack/httptools seems to be pretty similar and has recent commits, so you should probably switch over. Or fork / take ownership of http-parser

Reply:
> Not saying that you need to do it in this PR, but that project seems unmaintained. https://github.com/MagicStack/httptools seems to be pretty similar and has recent commits, so you should probably switch over. Or fork / take ownership of http-parser

Please tell that to `pymesos` developers :)
We don't use `http-parser` directly, it's a transitive dependency.